site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Report schedules'
+subhead: >
+  Learn when the Attribution Reporting API sends aggregatable and event-level reports.
+description: >
+  Learn when the Attribution Reporting API sends aggregatable and event-level reports.
+date: 2023-03-06
+updated: 2023-07-21
+authors:
+  - maudn
+---
+
+With the Attribution Reporting API, reports are not sent immediately after a user converts. Instead, the browser sends them with a delay.
+
+Note that regardless of report type, reports are only sent when the browser is running and online. After the first failure to send, the browser tries to send the report again after 5 minutes. After the second failure, the browser tries to send the report again after 15 minutes. After that, it's not sent.
+
+
+## Aggregatable reports
+
+{% Aside %}
+What's described in this section is the default behavior. If you need to filter out conversions happening after a certain window, you can use [custom report windows](/docs/privacy-sandbox/attribution-reporting/custom-report-windows/).
+{% endAside %}
+
+### Report delays
+
+Aggregatable reports are sent by the browser to the ad tech provider or advertiser with a random delay between zero and 10 minutes, or with a small delay after the browser starts again. 
+
+## Event-level reports
+
+{% Aside %}
+What's described in this section is the default behavior. If you need to customize report scheduling behavior by shortening a report window, or if you need to filter out conversions happening after a certain window, you can use [custom report windows](/docs/privacy-sandbox/attribution-reporting/custom-report-windows/).
+{% endAside %}
+
+View-through conversion reports are sent by the browser to the ad tech approximately one hour after the view event the conversion is attributed to is no longer eligible for attribution.
```

Comment:
I'm confused by this sentence—should it have been two?

Reply:
Clarified. thanks